packages/apollo-federation/src/service/printFederatedSchema.ts

```diff
@@ -295,7 +295,7 @@ function printArgs(args: GraphQLArgument[], indentation = '') {
   );
 }
 
-function printInputValue(arg: GraphQLArgument) {
+function printInputValue(arg: GraphQLInputField | GraphQLArgument) {
```

Comment:
How does this addition of `GraphQLInputField` come into play?

Reply:
Ah, I think I see now that this is related to:

https://github.com/apollographql/apollo-server/blob/cbdbe392964feeaa7882ef3535e92aebe8cc6791/packages/apollo-federation/src/service/printFederatedSchema.ts#L242

Though it's not immediately clear to me how it became problematic all of a sudden!